packages/vue-component/plugin/tag-handler.js

```diff
@@ -160,7 +160,7 @@ VueComponentTagHandler = class VueComponentTagHandler {
       }
 
       // Export
-      script = script.replace(jsExportDefaultReg, 'return');
+      script += `\nreturn require('${this.inputFile.getDisplayPath()}').default;`
```

Comment:
Well previously the code was just `return { … }`, so I think this is no different than the current behavior. I don’t think dynamic imports are currently supported with `vue-component`.